src/deprecated.jl

```diff
@@ -357,3 +357,32 @@ function by(d::AbstractDataFrame, cols::Any; sort::Bool=false, skipmissing::Bool
 end
 
 @deprecate groupvars(gd::GroupedDataFrame) groupcols(gd)
+
+@deprecate aggregate(d::AbstractDataFrame, fs::Any; sort::Bool=false) begin
+    df = select(d, names(d) .=> [fs])
+    sort && sort!(df)
+    return df
+end
+
+@deprecate aggregate(d::AbstractDataFrame, fs::AbstractVector; sort::Bool=false) begin
+    df = hcat([select(d, names(d) .=> [f]) for f in fs]..., makeunique=true)
```

Comment:
Could we check whether there are functions with duplicate names in `fs`? If not, then `select(d, names(d) .=> fs)` is enough right? Same for the GDF method.

Reply:
right - I will rewrite it.

Reply:
ah - it is not right, but I will write what is right.